src/stories/components/Teaser/Teaser.js

```diff
@@ -0,0 +1,130 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid } from '../../layouts/Grid/Grid';
+
+/**
+ * Component for teaser element.
+ *
+ * @component
+ * @param {string} image Source of the component.
+ * @param {string} url URL of the component.
+ * @param {string} title of the component.
+ * @param {string} description of the component.
+ * @param {number} portions of the component.
+ * @param {number} time in mins of the component.
+ * @param {number} rating of the component.
+ * @return {object} (
+ *   <Teaser url={url} image={image} eyebrow={eyebrow} title={title}
```

Comment:
In the teaser comment, change ```eyebrow``` to **```description```**, and ```portion``` to **```portions```** with an 's', to be consistent with props. 

Edit: Actually, change ```eyebrow``` to **```excerpt```** per the props passed at line 26?